assets/scss/templates/_signup.scss

```diff
@@ -6,7 +6,7 @@
   align-items: stretch;
 
   .main-panel {
-    flex: 1.5;
+    width: 500px;
```

Comment:
hmmm, this is not going to increase or decrease it size.... is this ok?

Reply:
Yes! This section no going to increase or decrease the size, maintain the same width and the breakpoint change to mobile